src/db/pool.cr

```diff
@@ -2,47 +2,73 @@ require "weak_ref"
 
 module DB
   class Pool(T)
-    @initial_pool_size : Int32
-    # maximum amount of objects in the pool. Either available or in use.
-    @max_pool_size : Int32
-    @available = Set(T).new
-    @total = [] of T
-    @checkout_timeout : Float64
-    # maximum amount of retry attempts to reconnect to the db. See `Pool#retry`.
-    @retry_attempts : Int32
-    @retry_delay : Float64
+    # Pool configuration
```

Comment:
It would be better to put comments describing ivar's in a separate line before them, like we do with other features like method.
Even if they're not documented, it's preferable to use as single style.

Reply:
done